Makefile

```diff
@@ -1,15 +1,144 @@
+
+## Python Variables
+# Python executable
+PYTHON                       := $(shell which python)
+# Poetry executable
+POETRY                       := $(shell which poetry)
+# Extract Python Version
+PYTHON_VERSION               ?= $(shell ${PYTHON} --version | cut -d ' ' -f 2 )
+# Temporary folder to output generated artifacts
+PYTHON_OUT_BIN_DIR           := ./dist
+
+# Docker compose specific environment configuration file
+DOCKER_COMPOSE_ENV_FILE      := .dbt.env
+
+## Github Variables
+# A github access token that provides access to upload artifacts under releases
+GITHUB_TOKEN                 ?= a_token
+# Github organization
+GITHUB_ORG                   := mattermost
+# Most probably the name of the repo
+GITHUB_REPO                  := ${APP_NAME}
+
+# ====================================================================================
+# Colors
+
+BLUE   := $(shell printf "\033[34m")
+YELLOW := $(shell printf "\033[33m")
+RED    := $(shell printf "\033[31m")
+GREEN  := $(shell printf "\033[32m")
+CYAN   := $(shell printf "\033[36m")
+CNone  := $(shell printf "\033[0m")
+
+# ====================================================================================
+# Logger
+
+TIME_LONG	= `date +%Y-%m-%d' '%H:%M:%S`
+TIME_SHORT	= `date +%H:%M:%S`
+TIME		= $(TIME_SHORT)
+
+INFO = echo ${TIME} ${BLUE}[ .. ]${CNone}
+WARN = echo ${TIME} ${YELLOW}[WARN]${CNone}
+ERR  = echo ${TIME} ${RED}[FAIL]${CNone}
+OK   = echo ${TIME} ${GREEN}[ OK ]${CNone}
+FAIL = (echo ${TIME} ${RED}[FAIL]${CNone} && false)
+
+# ====================================================================================
+# Verbosity control hack
+
+VERBOSE ?= 0
+AT_0 := @
+AT_1 :=
+AT = $(AT_$(VERBOSE))
+
+# ====================================================================================
+# Targets
+
+help: ## to get help
+	$(AT)echo "Welcome to ${APP_NAME}:${APP_VERSION}"
+	$(AT)echo "Usage:"
+	$(AT)grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) |\
+	awk 'BEGIN {FS = ":.*?## "}; {printf "make ${CYAN}%-30s${CNone} %s\n", $$1, $$2}'
+
+.PHONY: python-build
+python-build: ## to build python package
+	$(AT)$(INFO) python build $*...
+	$(AT)$(POETRY) build || ${FAIL}
+	$(AT)$(OK) python build $*
+
+.PHONY: python-test
+python-test: ## to run python tests
+	$(AT)$(INFO) testing python...
+	$(AT)$(POETRY) run pytest || ${FAIL}
+	$(AT)$(OK) testing python
+
+.PHONY: python-update-dependencies
+python-update-dependencies: ## to update python dependencies
+	$(AT)$(INFO) updating python dependencies...
+	$(AT)$(POETRY) install && \
+	    $(POETRY) run pip install clearbit==0.1.7 || ${FAIL}
+	$(AT)$(OK) updating python dependencies
+
+.PHONY: dbt-docs
+dbt-docs: ## to generate and serve dbt docs
+	$(AT)$(INFO) Generating docs and spinning up the a webserver on port 8081...
+	$(AT)export DBT_PROFILE_PATH=".";\
+	$(DOCKER) compose run -p "8081:8081" dbt_image bash -c "dbt deps && dbt docs generate -t prod && dbt docs serve --port 8081"
+
```

Comment:
Would expect an `$(AT)$(OK)  Generating docs and spinning up the a webserver on port 8081...` to provide proper logging

Reply:
The original intent was to leave it as it was (docker compose starting containers and waiting for `ctrl+c` to finish). Made a change to start the container in detached mode. Also added a (slightly) more helpful OK log message.